web/src/app/selection/MaterialSelect.tsx

```diff
@@ -164,11 +164,12 @@ export default function MaterialSelect(
       data-cy-ready={!isLoading}
       classes={customCSS}
       multiple={multiple}
-      filterSelectedOptions={multiple}
+      filterSelectedOptions
```

Comment:
This was necessary so that searching for a value in single-select mode won't omit the current one -- since there is no indicator that it's currently selected otherwise (in multiple, the chip is displayed).

Reply:
@mastercactapus I noticed that even though this was on for single items, it wasn't working as intended- the selected item would show with the default load, but if I typed in a search term, my selected item wouldn't be visible anymore. I made this change after I noticed my Cypress tests fail- so hopefully with this change it resolves that:

default when clicking on select (with GoAlert selected)

![Screen Shot 2022-12-05 at 9 19 20 AM](https://user-images.githubusercontent.com/11381794/205701341-3924b796-a40d-4ee5-82f2-86f83a85c498.png)


items after searching (with  GoAlert still selected)

![Screen Shot 2022-12-05 at 9 19 35 AM](https://user-images.githubusercontent.com/11381794/205701410-c77bf788-3095-4462-9e21-53d4b6fdf995.png)

Reply:
Talked offline- going to keep the filter as it was before and update the other tests that failed to reflect the new functionality.